packages/jest-worker/src/index.ts

```diff
@@ -45,6 +46,10 @@ function getExposedMethods(
   return exposedMethods;
 }
 
+export function serializerWithBigInt(data: unknown): unknown {
```

Comment:
this should not be in `index` to avoid circular dependencies. Please stick it into a `utils`, `serializers` or something